internal/pkg/cli/task_run.go

```diff
@@ -371,6 +379,42 @@ func (o *runTaskOpts) Validate() error {
 		}
 	}
 
+	if len(o.secrets) > 0 {
+		if err := o.validateSecrets(); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (o *runTaskOpts) validateSecrets() error {
+	if o.appName == "" && o.env == "" && !o.acknowledgeSecretsAccess {
+		fmt.Println("Looks like you're requesting ssm:GetParameters to the following SSM parameters:")
+
+		for _, value := range o.secrets {
+			if !arn.IsARN(value) || strings.Contains(value, ":ssm:") {
+				fmt.Println("* " + value)
+			}
+		}
+
+		fmt.Println("\nand secretsmanager:GetSecretValue to the following Secrets Manager secrets:")
+
+		for _, value := range o.secrets {
+			if strings.Contains(value, ":secretsmanager:") {
+				fmt.Println("* " + value)
+			}
+		}
+
+		v, err := o.prompt.Confirm(taskSecretsPermission, "")
```

Comment:
We can provide some help message here to explain 
1. Why we need the permissions
2. The result of confirming on this question.